client/src/components/pages/MainChannel.tsx

```diff
@@ -219,6 +219,7 @@ const Screen: FC = () => {
         <ContentContainer searchArgs={searchArgs} />
       </Suspense>
       <TouchableOpacity
+        testID="channel-create-FAB"
```

Comment:
I'd rather change this to `channel-create-fab`. The naming convention should be one of those [listed here](https://dooboo-ui.dooboolab.com/?path=/docs/contributing-contributing--page#naming-conventions).